pages/dashboard/v2/details/[id]/[studentEmail].js

```diff
@@ -51,19 +60,47 @@ export async function getServerSideProps(context) {
     return {};
   }
 
+  const certificationNumbers = await prisma.classroom.findUnique({
+    where: {
+      classroomId: context.params.id
+    },
+    select: {
+      fccCertifications: true
+    }
+  });
+
+  let superblockURLS = await getDashedNamesURLs(
+    certificationNumbers.fccCertifications
+  );
+
+  let superblockTitleInfo = await getNonDashedNamesURLs(
+    certificationNumbers.fccCertifications
+  );
+  let superBlockJsons = await getSuperBlockJsons(superblockURLS);
+  let dashboardObjs = createDashboardObject(superBlockJsons);
+
+  // console.log('DDASHBOARD OBJS ===>', dashboardObjs)
+  let studentData = await getIndividualStudentData(studentEmail);
```

Comment:
ISR should be used for the student data API response. This will allow us to avoid merging the data, and simply update the respective UI elements.